apps/wallet/src/ui/app/shared/faucet/request-button/index.tsx

```diff
@@ -26,18 +28,26 @@ function FaucetRequestButton({
     const network = useAppSelector(({ app }) => app.apiEnv);
     const networkName = API_ENV_TO_INFO[network].name;
     const showFaucetRequestButton = API_ENV.customRPC !== network;
-    const dispatch = useAppDispatch();
-    const loading = useAppSelector(({ faucet }) => faucet.loading);
+    const mutation = useFaucetMutation();
+    const isMutating = useIsFaucetMutating();
```

Comment:
Why not just get this from the `useFauceMutation`?

Reply:
Good point, I'll change it

Reply:
Done